sycl/test-e2e/Matrix/SG32/joint_matrix_bfloat16.cpp

```diff
@@ -5,19 +5,49 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
 // UNSUPPORTED: gpu-intel-dg2
-
 // REQUIRES: target-spir
 
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+// RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=2 %{run} %t.out %}
+// RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=1 %{run} %t.out %}
+// RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=0 %{run} %t.out %}
 
 #include "common.hpp"
 
 #define SG_SZ 32
 
-#include "joint_matrix_bfloat16_impl.hpp"
+#include "joint_matrix_16bit_impl.hpp"
```

Comment:
what is tested before and after this change is not equivalent. Could you update the code so that old test coverage would be not less than before.